subsys/bluetooth/audio/ascs.c

```diff
@@ -354,6 +356,29 @@ static void ase_exit_state_streaming(struct bt_ascs_ase *ase)
 	} else {
 		LOG_WRN("No callback for stopped set");
 	}
+	if (next_state == BT_BAP_EP_STATE_QOS_CONFIGURED && ase->ep.dir == BT_AUDIO_DIR_SINK) {
+		if (ops != NULL && ops->disabled != NULL) {
+			ops->disabled(stream);
+		} else {
+			LOG_WRN("No callback for disabled set");
+		}
+	}
+}
+
+static void ase_exit_state_enabling(struct bt_ascs_ase *ase)
+{
+	struct bt_bap_stream *stream = ase->ep.stream;
+	struct bt_bap_stream_ops *ops;
+	const enum bt_bap_ep_state next_state = ascs_ep_get_state(&ase->ep);
+
+	ops = stream->ops;
+	if (next_state == BT_BAP_EP_STATE_QOS_CONFIGURED && ase->ep.dir == BT_AUDIO_DIR_SINK) {
```

Comment:
```suggestion
	ops = stream->ops;
	
	if (next_state == BT_BAP_EP_STATE_QOS_CONFIGURED && ase->ep.dir == BT_AUDIO_DIR_SINK) {
```

Please also add a comment why we do this, as this implements functionality that differs from the state machine